examples/lighting-app/nrfconnect/rpc.overlay

```diff
@@ -15,9 +15,7 @@
 #
 
 # This file should be used as a configuration overlay to build Pigweed RPCs to
-# lighting-app:
```

Comment:
why removed?

Reply:
Because it duplicates the information from README.md: https://github.com/project-chip/connectedhomeip/blob/master/examples/lighting-app/nrfconnect/README.md#building-with-pigweed-rpcs and last time I forgot to modify it, so I thought it might be good to keep it only in the one place.